src/main/java/com/labsynch/labseer/service/StandardizationServiceImpl.java

```diff
@@ -731,7 +731,11 @@ public StandardizationHistory getMostRecentStandardizationHistory() {
 		List<StandardizationHistory> standardizationSettingses = StandardizationHistory
 				.findStandardizationHistoryEntries(0, 1, "id", "DESC");
 		if (standardizationSettingses.size() > 0) {
-			return standardizationSettingses.get(0);
+			StandardizationHistory mostRecentHistory = standardizationSettingses.get(0);
+			// Refresh the entity to ensure it is up-to-date
+			EntityManager em = StandardizationHistory.entityManager();
```

Comment:
This makes sure the we are doing a round trip to the database and that we aren't working with a cached version of the entity.